iOS-Email-Client/AsyncTasks/SendMailAsyncTask.swift

```diff
@@ -38,7 +38,7 @@ class SendMailAsyncTask {
         self.criptextEmails = recipients.1
         self.files = files
         self.emailRef = DBManager.getReference(email)
-        self.fileKey = fileKey
+        self.fileKey = files.count > 0 ? fileKey : nil
```

Comment:
isn't finding a `fileKey` in the DB, enough? why not `self.fileKey = DBManager.getFileKey(emailId: email.key)?.key`?